packages/easy-testing/src/Command/ValidateFixtureSkipNamingCommand.php

```diff
@@ -56,11 +56,22 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return ShellCode::SUCCESS;
         }
 
-        foreach ($missplacedFixtureFileInfos as $missplacedFixtureFileInfo) {
-            if (Strings::match(
+        foreach ($missplacedFixtureFileInfos as $key => $missplacedFixtureFileInfo) {
+            $isSkipped = (bool) Strings::match(
                 $missplacedFixtureFileInfo->getBasenameWithoutSuffix(),
                 Prefix::SKIP_PREFIX_REGEX
-            )) {
+            );
+            $hasSplit = strpos(
+                (string) $this->smartFileSystem->readFile((string) $missplacedFixtureFileInfo),
+                '-----'
+            ) !== false;
+
+            if ($isSkipped && $hasSplit) {
+                unset($missplacedFixtureFileInfos[$key]);
+                continue;
+            }
+
+            if ($isSkipped) {
                 // A. file has incorrect "skip"
```

Comment:
This command should only report the result. The logic that decides what happens should be here:
This is already checked here: https://github.com/symplify/symplify/blob/b7d17fdf5759748c4e78f650f9477f646c117e15/packages/easy-testing/src/MissplacedSkipPrefixResolver.php#L27-L44

Maybe returning 2 items (one with own method) would be better:

- incorrect skips 
- missing skips

Then each would be foreached separatedly here.

Reply:
I will try.

Reply:
implemented.

Reply:
Thank you :+1: